pkg/app/metadata.json

```diff
@@ -0,0 +1,965 @@
+        "title": "Update multiple ACL entries using command substitution"
+      }],
+      "apis": [
+        "https://developer.fastly.com/reference/api/acls/acl-entry/#bulk-update-acl-entries",
+        "https://developer.fastly.com/reference/api/acls/acl-entry/#update-acl-entry"
+      ]
+    }
+  },
+  "auth-token": {
+    "create": {
+      "apis": ["https://developer.fastly.com/reference/api/auth/#create-token"]
+    },
+    "delete": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/auth/#revoke-token-current",
+        "https://developer.fastly.com/reference/api/auth/#bulk-revoke-tokens",
+        "https://developer.fastly.com/reference/api/auth/#revoke-token"
+      ]
+    },
+    "describe": {
+      "apis": ["https://developer.fastly.com/reference/api/auth/#get-token-current"]
+    },
+    "list": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/auth/#list-tokens-customer",
+        "https://developer.fastly.com/reference/api/auth/#list-tokens-user"
+      ]
+    }
+  },
+  "backend": {
+    "create": {
+      "examples": [{
+        "cmd": "fastly backend create --name example --address example.com --version latest",
+        "description": "Create a backend with a hostname assigned to the --address flag. The --override-host, --ssl-cert-hostname and --ssl-sni-hostname will default to the same hostname assigned to --address.",
+        "title": "Create a backend on a Fastly service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/services/backend/#create-backend"]
+    },
+    "delete": {
+      "examples": [{
+        "cmd": "fastly backend delete --name example --version latest",
+        "title": "Delete a backend on a Fastly service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/services/backend/#delete-backend"]
+    },
+    "describe": {
+      "examples": [{
+        "cmd": "fastly backend describe --name example --version latest",
+        "title": "Show detailed information about a backend on a Fastly service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/services/backend/#get-backend"]
+    },
+    "list": {
+      "examples": [{
+        "cmd": "fastly backend list --version latest",
+        "title": "List backends on a Fastly service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/services/backend/#list-backends"]
+    },
+    "update": {
+      "examples": [{
+        "cmd": "fastly backend update --name example --new-name testing --version latest",
+        "title": "Update a backend on a Fastly service version"
+      }],
+      "apis": ["https://developer.fastly.com/reference/api/services/backend/#update-backend"]
+    }
+  },
+  "compute": {
```

Comment:
The build, deploy and publish examples are very basic but seemed like a reasonable excuse to highlight specific flags like `--skip-verification` and `--accept-defaults`. Same comment for the serve example which really is so basic but was a chance to highlight the `--watch` flag.